src/ome_zarr_models/_rfc5_transforms/multiscales.py

```diff
@@ -0,0 +1,247 @@
+from __future__ import annotations
+
+from typing import Self
+
+from pydantic import (
+    BaseModel,
+    Field,
+    JsonValue,
+    field_validator,
+    model_validator,
+)
+
+from ome_zarr_models._rfc5_transforms.coordinate_transformations import (
+    CoordinateSystem,
+    CoordinateTransformationType,
+    Scale,
+    Translation,
+)
+from ome_zarr_models.base import BaseAttrs
+from ome_zarr_models.common.validation import check_length
+
+__all__ = ["Dataset"]
+
+
+class Dataset(BaseAttrs):
+    """
+    An element of Multiscale.datasets.
+    """
+
+    path: str
+    # TODO: suggest John not to have a tuple for the coordinate transformations of a
+    #  dataset, since it's either a single scale or a single sequence (with a scale
+    #  and a translation)
+    coordinateTransformations: tuple[CoordinateTransformationType, ...] = Field(
+        ..., min_length=1, max_length=1
+    )
+
+    # the before validation is used to simplify the error messages
+    @field_validator("coordinateTransformations", mode="before")
+    def _ensure_scale_translation(
```

Comment:
this method probably needs to be renamed

Reply:
Any suggestion? A more explicit name could be `_ensure_scale_or_sequence_of_scale_translation()`. But maybe the current one is good enough (and consistent with `common.multiscale.Dataset`).

Reply:
i don't think scale + translation is [required any more](https://github.com/bogovicj/ngff/blob/3822dd0d9d2e388a12b6b74d0bffc7a215298e42/0.6-dev/index.bs#L1348-L1351), so maybe you can get rid of this method entirely?

Reply:
But so here now we could have an affine or other things? If this is the case, then a `Multiscale` object would go beyond a pyramidal image. It could represent a tiled image, or a set of 2 images in a 3D space, etc.

Is this what the new specs want?

Reply:
Opened an issue for this here: https://github.com/ome/ngff/issues/331